No python generator support for properties with interpolation. Inflectors cause misidentification as union.

The use of interpolated parameters as in the following sample appears to be unsupported.  Is this on the roadmap?

```
resourceTypes:
  collection:
    get:
      responses:
        200:
          body:
            application/json:
              properties:
                <<resourcePathName|!pluralize>>: <<resourcePathName|!uppercamelcase|!singularize>>[]

```

Interestingly the generator reports an error `validator has no support for union, ignore it`.  The python field generator searches for "|" in the type and identifies it as a union.

Unfortunately, no.  The issue seems to be that there is no parameter interpolation for these elements.  It also affects the go-generator but with a somewhat less ambiguous error.

> Error running go fmt on 'outgo/UsersGetRespBody.go' failed:
> outgo/UsersGetRespBody.go:11:9: expected '}', found '<<'
> outgo/UsersGetRespBody.go:25:5: expected declaration, found 'return'

The generate code clearly shows the raw parameter name:

```
type UsersGetRespBody struct {

        <<ResourcePathName|!Pluralize>>   []interface{} `json:"<<resourcePathName|!pluralize>>" validate:"nonzero"`
}
```

--- a/raml/method.go
+++ b/raml/method.go
@@ -358,7 +358,9 @@ func (b *Bodies) inherit(parent Bodies, dicts map[string]interface{}) {
 				case strings.HasSuffix(k, "?"): // if only ended with `?`, we can ignore it
 					continue
 				}
-				b.ApplicationJSON.Properties[k] = p
+				k = substituteParams(k, k, dicts)
+				prop := ToProperty(k, p)
+				b.ApplicationJSON.Properties[k] = substituteParams(prop.Type, prop.Type, dicts)
 			}
 		}
 	}

